src/parser/GenerateOperations.js

```diff
@@ -0,0 +1,78 @@
+// eslint-disable-next-line sonarjs/cognitive-complexity
+function operationsFromChannels (channels,PublishOperations,SubscribeOperations) {
```

Comment:
Why do you change between camel-cased variables and parameters to pascal-cased? 🤔 

- https://github.com/asyncapi/fluffy-robot/pull/39/files#r687726334